packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -53,6 +55,35 @@ const BaseChart = ({ viewport, queries, size = { width: 500, height: 500 }, ...o
 
   const { group } = useViewport();
 
+  const setTcs = useTrendCursorsStore((state) => state.set);
+  // const addTc = useTrendCursorsStore((state) => state.add);
+  useEffect(() => {
+    setTcs(([]))
+  }, [])
+  const trendCursors = useTrendCursorsStore((state) => state.trendCursors);
+  useEffect(() => {
+    chartRef.current?.setOption({
+      trendCursors
+    },
+    {
+      replaceMerge: ['trendCursors']
+    })
+  }, [trendCursors]);
+  useEffect(() => {
+    chartRef.current?.getZr().on('click', (e) => {
```

Comment:
Does this effect need a return to unsubscribe from the event to prevent build up of handlers?